build-tools/common/src/main/kotlin/aditogether/buildtools/utils/PropertiesUtils.kt

```diff
@@ -0,0 +1,27 @@
+package aditogether.buildtools.utils
+
+import org.gradle.api.Project
+
+/**
+ * Returns the Gradle property with the name [name] in a string format.
+ * This throws if the property is not present.
```

Comment:
Nit: what about `@throws AbcException` here and below?

Reply:
Sure! Done here: https://github.com/androiddevelopersitalia/aditogether/pull/66/commits/0dce2c2571525e5fde952f6f22febd3f6db71fba